site/en/docs/android/trusted-web-activity/quick-start/index.md

```diff
@@ -34,15 +34,13 @@ you're using a physical phone).
 A Trusted Web Activity lets your Android App launch a full screen Browser Tab without
 any browser UI.
 This capability is restricted to websites that you own, and you prove this by setting
-up Digital Asset Links.
-Digital Asset Links consist essentially of a file on your website that points to your app and some
-metadata in your app that points to your website.
-We'll talk [more about them later](#a-note-on-signing-keys).
+up Digital Asset Links. We'll talk [more about them later](#creating-your-asset-link-file).
+
 
 When you launch a Trusted Web Activity, the browser will check that the Digital Asset Links check
 out, this is called **verification**.
 If verification fails, the browser will fall back to displaying your website as a
-[Custom Tab](/docs/android/custom-tabs).
+[Custom Tab](/docs/android/custom-tabs/overview/).
```

Comment:
I'm happy you're fixing this link but this feels like a failure: there should be a page at /docs/android/custom-tabs instead, but it 404's right now.